keylime/elchecking/example.py

```diff
@@ -111,12 +127,20 @@ def refstate_to_test(self, refstate: policies.RefState) -> tests.Test:
         vd_authority = tests.VariableDispatch()
 
         def bsa_test(kernel: typing.Dict[str, str]) -> tests.Test:
+            if not has_secureboot:
```

Comment:
From where `has_secureboot` comes from? Maybe you want a new field `self.has_secureboot` set early in `refstate_to_test`?

Reply:
It is set early in the `refstate_to_test` method:
https://github.com/keylime/keylime/blob/1266fde473f721b432dd090da421fe86cd6dc19b/keylime/elchecking/example.py#L94

Reply:
I saw that, but I did not realize that those are local function that close over this!

Sorry for the noise!